app/views/PillarAnalysis/AnalyticalStatementInput/AnalyticalEntryInput/index.tsx

```diff
@@ -53,7 +55,15 @@ interface AnalyticalEntryInputProps {
         droppedValue: DroppedValue,
         dropOverEntryClientId: string | undefined,
     ) => void;
+    onAnalyticalEntryDown: (
+        dropOverEntryClientId: string | undefined,
+    ) => void;
+    onAnalyticalEntryUp: (
+        dropOverEntryClientId: string | undefined,
+    ) => void;
```

Comment:
dropOverEntryClientId should be required, because without it this function doesn't make sense.